src/assignement2/NewUser.java

```diff
@@ -0,0 +1,68 @@
+package assignement2;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+/*
+ * This class is inputing all the parameter required
+ * for the User
+ */
+
+public class NewUser {
+
+    /*
+     * method is taking input of name
+     * roll no
+     * age
+     * address
+     * and the course he choices
+     * @return user class
+     */
+
+    public static User getNewUser() {
+        final Scanner sc = new Scanner(System.in);
+        ArrayList<String> subj = new ArrayList<>();
+        final int maxSub = Constants.ch_5;
```

Comment:
why are u assinging a constant to a variable ? cant u use it directly

Reply:
thought of it and made the changes